aws/apigateway/enable-access-logging/metadata.rego

```diff
@@ -0,0 +1,17 @@
+package cloud-metadata.aws.apigateway.enable-access-logging
+
+__rego_metadata__ := {
+	"id": "AWS061",
+	"apiVersion": 2,
+	"version": 1,
+	"title": "API Gateway stages for V1 and V2 should have access logging enabled",
```

Comment:
Are we going to create a standards document before we start this governing our use of MUST, SHOULD etc.? e.g. https://www.ietf.org/rfc/rfc2119.txt

Reply:
Once this merge https://github.com/aquasecurity/appshield/pull/125/files#diff-eb28ee6e2c7606c807df369a121e13add898e6bf81050206c327ef86df3b25e5 we can have reference to appshield repo

Reply:
That looks like a great starting point, but I think we also need to state what wording we should use in the titles for consistency e.g. "Bucket should be encrypted" vs "Bucket must be encrypted"

Reply:
@joshlarsen Any preferences?

Reply:
@oranmoshai @liamg definitely prefer "Bucket should be encrypted." We can say that the best practice and recommendation is that buckets **should** be encrypted. We don't have any say over what the organization's actual policy is.